src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -61,12 +61,15 @@ export class GraphStyles {
       return ele.cy().scratch(CytoscapeGlobalScratchNamespace);
     };
 
-    const getEdgeColor = (ele: any): string => {
-      const rate = ele.data('rate') ? Number(ele.data('rate')) : 0;
-      if (rate === 0 || ele.data('isUnused')) {
+    const getHttpEdgeColor = (ele: any): string => {
+      if (ele.data('isUnused')) {
         return EdgeColorDead;
       }
-      const pErr = ele.data('percentErr') ? Number(ele.data('percentErr')) : 0;
+      const http = ele.data('http') ? Number(ele.data('http')) : 0;
+      if (http === 0 || ele.data('isUnused')) {
```

Comment:
if ele.data('isUnused'), you will have already returned "EdgeColorDead" on line 66. I don't think you need the right side of this || expression.

Reply:
OK, I'll fix that